### DISCUSSION TITLE
Permanent install

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#335] [Q&A] [answered]
Sorry for the obtuse title, but this might be a bit unusual.

I set EOS up on a Samsung USB/SSD for my desktop a while back. I've been using it (and tweaking) for a while and it works well.

The desktop has Win10 installed on the internal drive.

I've gotten to the point where I would like to migrate the Samsung installation to the internal drive.

Any thoughts on the simplest way to do that?

Thanks

**Comment by secondlnl** [comment]
As a first thought I would swap the two drives if possible.

**Comment by RogerWeihrauch** [comment]
hi
hm, if you can absolutely throw windows into the bin, I would suggest a clean an full new install.
Sure, all the changes are also gone, then. And it takes much more time. But, maybe you copied all the steps you made in there into a text file as backup?  ;)
This will be in my eyes the most reliable way to have a smooth and clean running system.
Regards
Roger

**Comment by doobes** [comment]
Well that turned out to be a pain.

CloneZilla absolutely refused to write to the fixed SSD for a plethora of reasons.

So, I took the drive out of the box, connected it to a USB interface and then used the free version of Macrium Reflect on a Windows box to clone the Samsung SSD to it.

Putting the drive back in the box, it fired right up.

Now I have to figure out how to tell Grub that there isn't a Windows 10 installation to boot.

**Comment by RogerWeihrauch** [comment, marked as answer]
You can tell Linux which ‘kernel’ to boot with follwing entries in GRUB:

GRUB_SAVEDEFAULT=true
GRUB_DEFAULT=saved

This keeps the last kernel selection in mind.

Or/and additionally to deactivate the OS scan function in GRUB:

#GRUB_DISABLE_OS_PROBER=false

By uncommenting this line

Don’t forget to do an ‘update-grub2’ afterwards.

Regards,
Roger

**Reply by doobes** [reply]
Thank you

**Comment by RogerWeihrauch** [comment]
U‘r welcomeRegards,RogerOn Oct 16, 2023, at 5:48 PM, doobes ***@***.***> wrote:﻿
Thank you
